docs/ui/cacao/advanced_cacao_cli.md

```diff
@@ -0,0 +1,98 @@
+![cacao logo](images/cacao-logo.png){ width=128px }[CACAO Overview](overview.md) &gg; Using the CACAO CLI
+
+# CACAO Command-Line Interrface
+
+The CACAO Command-Line Interface (CLI) allows command-line users to interact with CACAO. The CLI is a Go program that can be installed on any system.
+
+## Installation
+
+1. The CACAO CLI is a single binary that can be installed on any system. The most recent cli binaries can be downloaded from [the CACAO packages page on Gitlab](https://gitlab.com/cyverse/cacao/-/packages) or you can download from here:
+
+- [CACAO Darwin ARM64](https://gitlab.com/cyverse/cacao/-/package_files/101908036/download)
+- [CACAO Darwin AMD64](https://gitlab.com/cyverse/cacao/-/package_files/101907995/download)
```

Comment:
```suggestion
- [CACAO Mac (Darwin) ARM64](https://gitlab.com/cyverse/cacao/-/package_files/101908036/download)
- [CACAO Mac (Darwin) AMD64](https://gitlab.com/cyverse/cacao/-/package_files/101907995/download)
```

Readers are more likely to know that they have a Mac, than what Darwin is in this context